rpcs3/Emu/Cell/Modules/cellGame.cpp

```diff
@@ -671,6 +671,9 @@ error_code cellGameContentPermit(vm::ptr<char[CELL_GAME_PATH_MAX]> contentInfoPa
 		temp.file.write(psf::save_object(perm.sfo));
 		ensure(temp.commit());
 
+		// Flush all pending file buffers to disk before commitment (TODO: Win32)
+		fs::sync();
```

Comment:
I think, remove it for now, even if it works it may take extreme amount of time.

Reply:
okay.